core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -73,14 +73,6 @@ oppia.directive('audioBar', [
               $scope.languagesInExploration.length > 0);
           };
 
-          $scope.getProgressBarMode = function() {
-            if ($scope.audioLoadingIndicatorIsShown) {
-              return 'indeterminate';
```

Comment:
Just to check: what happened to the indeterminate loading case? /cc @tjiang11

Reply:
@seanlip I missed this part. In the follow-up PR #4670 I have added a progress bar only if audio is still loading. In the case where audio has loaded, a slider will be displayed. I am not sure about this approach. I would love to hear suggestions on this!

Reply:
That sounds reasonable, thanks. I'll pass that PR to @tjiang11 for review.

Reply:
Thanks @seanlip!